app/src/main/java/org/piramalswasthya/sakhi/di/AppModule.kt

```diff
@@ -28,10 +28,19 @@ import javax.inject.Singleton
 @InstallIn(SingletonComponent::class)
 object AppModule {
 
+//     Staging Base URL
     private const val baseTmcUrl =
         "https://amritdemo.piramalswasthya.org/"
 
-    private const val baseAbhaUrl = "https://healthidsbx.abdm.gov.in/api/"
```

Comment:
What will happen when we deploy to prod?
Should this URl be changed before building? Can we pick up the URL from build config?

Reply:
Yes, we have to make changes for that, will do that.